action.yaml

```diff
@@ -1,11 +1,26 @@
 
 name: 'Qodana - Code Inspection'
-description: 'Qodana is a code quality monitoring tool to identify and suggest fixes for bugs, security vulnerabilities, duplications, and imperfections.
-It brings all the smart features you love in the IntelliJ IDEs into your project pipelines.'
+description: 'Qodana is a code quality monitoring tool that identifies and suggests fixes for bugs, security vulnerabilities, duplications, and imperfections.'
 author: 'JetBrains'
 
+inputs:
```

Comment:
Maybe something like this?
```yaml
  project-dir:
    description: 'Project folder to inspect  (default: whole project)'
    required: false
    default: '/github/workspace'
  results-dir:
    description: 'Save results to folder (default: qodana)'
    required: false
    default: 'qodana'
  report-dir:
    description: 'Save html report to folder (default: report)'
    required: false
    default: 'report'

runs:
  using: 'docker'
  image: 'docker:://jetbrains/qodana'
  args:
    - --project-dir=${{inputs.project-dir}}
    - --results-dir=${{inputs.project-dir}}/${{inputs.results-dir}}
    - --report-dir=${{inputs.project-dir}}/${{inputs.reports-dir}}
    - --save-report
```
If there would be request to disable `--save-report` we will consider reading `inputs` from ENV.

Reply:
Looks good